### PR TITLE
Add keyboards Dactyl `lightcycle` and `maximus`

### PR DESCRIPTION
**Pull request by dunk2k**
## Description

Add boards `dactyl_lightcycle` and `dactyl_maximus`

## QMK Pull Request 

Dactyl Lightcycle: [qmk/qmk_firmware#20709](https://github.com/qmk/qmk_firmware/pull/20709)  
Dactyl Maximus: [qmk/qmk_firmware#21338](https://github.com/qmk/qmk_firmware/pull/21338) 

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Dactyl Lightcycle",
+  "vendorId": "0x444D",
```
Comment (Xelus22) [inline v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json:3]: in QMK source this is 0xFEED.

File: v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  "productId": "0x3536",
```
Comment (Xelus22) [inline v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json:4]: In QMK soruce this is 0x3060

**Review by Xelus22 — CHANGES_REQUESTED** [review]
https://www.caniusevia.com/docs/layouts#kle-json-rules

Please colour code the keys.

File: v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json (hunk 1 above)
Comment (dunk2k) [reply v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json:3]: [qmk/qmk_firmware#21462](https://github.com/qmk/qmk_firmware/pull/21462) corrects this

File: v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json (hunk 2 above)
Comment (dunk2k) [reply v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json]: amended as of commit `0cfd8bb`

**Comment by dunk2k** [comment]
> Please colour code the keys.

1. As these layouts are not conventional (i.e. do not have designate modifier, alpha, numpad, navigation positions), assigning a position as modifier/alpha would invalidate the layout's flexible and not conventional raison d'etre
2. Other non-conventional layout keyboards do not have colour codes assigned. Examples:
    - crkbd/crkbd.json
    - handwired/dactyl_manuform/5x6_5/5x6_5.json
    - maplecomputing/jnao.json

File: v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json (hunk 1 above)
Comment (Xelus22) [reply v3/handwired/dactyl_lightcycle/dactyl_lightcycle.json:3]: ^ will wait on this before merging 👍

File: v3/handwired/dactyl_maximus/dactyl_maximus.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
+{
+  "name": "Dactyl Maximus",
+  "vendorId": "0x444D",
+  "productId": "0x3536",
```
Comment (Xelus22) [inline v3/handwired/dactyl_maximus/dactyl_maximus.json:4]: image

github actions is flagging this as a duplicate VID/PID combo.

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/handwired/dactyl_maximus/dactyl_maximus.json (hunk 3 above)
Comment (dunk2k) [reply v3/handwired/dactyl_maximus/dactyl_maximus.json]: Created [corrective PR](https://github.com/qmk/qmk_firmware/pull/21666) in qmk/qmk_firmware for `dactyl_maximus` VID and PID

**Comment by dunk2k** [comment]
Quick update: Both correction PRs have been merged in qmk/qmk_firmware

**Review by Xelus22 — APPROVED** [review]

**Comment by dunk2k** [comment]
@Xelus22 Thank you kindly

**Pull request merged**